### PR TITLE
Add "k4run" gaudirun script (formerly fccrun)

### PR DESCRIPTION
**Pull request by vvolkl**
This exposes almost all job options as command line parameters.

### CODE REVIEW

File: FWCore/scripts/k4run [hunk 1]
```diff
@@ -63,8 +64,8 @@ if __name__ == "__main__":
     # print a doc line showing the configured algorithms
     if run_with_options_file:
       for alg in ApplicationMgr().TopAlg:
-        print ' --> ', alg.name(), 
-      print "",'\n'
+        print(' --> ', alg.name(), )
```
Comment (andresailer) [inline FWCore/scripts/k4run:67]:
```suggestion
        print(' --> ', alg.name(), end='')
```
or does the comma at the end also prevent a new line with the print function?

**Comment by mirguest** [comment]
Hi Valentin,
Is it possible to specify the evtmax in this script? 

Thank you.
Tao

File: FWCore/scripts/k4run (hunk 1 above)
Comment (vvolkl) [reply FWCore/scripts/k4run]: It does not, thanks!

**Comment by vvolkl** [comment]
Hi Tao, yes it is, with `k4run -n EVTMAX`, see https://github.com/key4hep/K4FWCore/blob/a933d5bce483b25cb3795ee1512c287820e15cff/FWCore/scripts/k4run#L46

**Comment by mirguest** [comment]
> Hi Tao, yes it is, with `k4run -n EVTMAX`, see
> 
> https://github.com/key4hep/K4FWCore/blob/a933d5bce483b25cb3795ee1512c287820e15cff/FWCore/scripts/k4run#L46

Hi Valentin, it is great. Thanks.

**Review by andresailer — APPROVED** [review]
Looks good to me

**Pull request merged**